render/README.md

```diff
@@ -1,37 +1,50 @@
 # Unimarkup Render
 
-This crate defines types and traits used in the [inline](../inline/README.md) and [core](../core/README.md) crate to render Unimarkup elements to all supported output formats.
-Therefore, all Unimarkup elements must implement the [`Render`](src/render.rs) trait defined in this crate.
+This crate contains traits and implementations to render Unimarkup elements to all supported output formats.
+The [`Render`](src/render.rs) trait defined in this crate is used to implement rendering to an output format.
+It is already implemented for all supported formats, but it may be used to add support for other formats.
```

Comment:
Maybe extend a little: 

```suggestion
It is already implemented for all supported formats, but it may be used to add support for other formats, or for alternative rendering implementations. 
```